assignments/arrays.js

```diff
@@ -63,34 +63,69 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(inventory[32]);
```

Comment:
This will give us the data that we want but not in the proper syntax. We would rather access the individual properties on the objects and have it look a little cleaner. The easiest and cleanest way is with template literals. These are just a fancy syntax that allows us to access properties of JS objects within our console statements. For example:
```console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`)```
A couple of things to note about the syntax. Everything in the console log is in backticks (not single quotes) in order to allow us to access these properties. We then just use the ${} syntax with whatever property we want to reference inside the curly braces. Again these are called template literals and make it very easy to access properties later without having to assign them to variables.